package.json

```diff
@@ -50,7 +50,7 @@
     "dotenv": "^16.0.2",
     "echarts": "^5.5.0",
     "echarts-for-react": "^3.0.2",
-    "ethers": "^5.7.1",
+    "ethers": "^5",
```

Comment:
Do we need to change this dependency globally? Just wondering if this will break something elsewhere?

Reply:
no need, I will change it to the old value